kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementContainmentControl.java

```diff
@@ -109,6 +109,7 @@ public boolean onMove(double dx,
 
     @Override
     public boolean onMoveComplete() {
+        state.getGhost().ifPresent(WiresShape::removeFromParent);
```

Comment:
@jeremyary This line puts the child element under the right stage after drag/drop.